src/main/java/roomescape/reservation/service/ReservationService.java

```diff
@@ -73,8 +101,8 @@ public List<MyReservationResponse> findReservationsByMember(Long memberId) {
         Member member = memberRepository.findById(memberId)
             .orElseThrow(MemberNotExistsException::new);
 
-        return reservationRepository.findByMember(member).stream()
-            .map(MyReservationResponse::new)
+        return reservationRepository.findWithRankByMember(memberId).stream()
+            .map(MyReservationResponse::from)
             .collect(Collectors.toList());
```

Comment:
```suggestion
        return reservationRepository.findWithRankByMember(memberId).stream()
            .map(MyReservationResponse::from)
            .toList();
```
> "Stream.toList()" method should be used instead of "collectors" when unmodifiable list needed

소나린트 경고가 발생하네요 ㅎㅎ 전체적으로 `.collect(Collectors.toList())` 선언한 부분을 위 처럼 변경해도 되겠어요~!